package.json

```
@@ -60,10 +75,14 @@
     "husky": "^6.0.0",
     "jest": "^27.2.0",
     "lint-staged": "^10.5.4",
+    "npm-run-all": "^4.1.5",
+    "npm-watch": "^0.11.0",
     "prettier": "^2.2.1",
     "semantic-release": "^19.0.2",
     "ts-jest": "^27.0.5",
     "ts-node": "^10.2.1",
+    "typedoc": "^0.23.24",
+    "typedoc-plugin-markdown": "^3.14.0",
```

```suggestion
    "typedoc-plugin-markdown": "^4.4.2",
```

https://github.com/TypeStrong/typedoc/issues/2615